Wikipedia/Code/VanishAccountPopUpAlertView.swift

```diff
@@ -47,17 +69,32 @@ struct VanishAccountPopUpAlertView: View {
                                     .frame(maxWidth: .infinity)
                                     .frame(height: 43, alignment: .center)
                                     .foregroundColor(Color(theme.colors.link))
-                                    .font(Font(titleFont))
+                                    .font(Font(boldFont))
                             }).buttonStyle(PlainButtonStyle())
+                                .frame(height: 43)
                         }
-                        .frame(width: 300)
+                        .frame(width: 300, height: 470)
```

Comment:
I notice the text cuts off with dynamic type increased now on the popup. I suspect it might be this line, when you limit it to a particular height. This is a screenshot from iPhone 7, iOS 15:

![IMG_0756](https://user-images.githubusercontent.com/3620196/187554778-fd80a437-eae3-410e-9425-db8fa77a1e93.PNG)